integration_tests/integration/court-register.spec.ts

```diff
@@ -19,13 +19,15 @@ context('Court register', () => {
         courtName: 'Sheffield Crown Court',
         courtDescription: 'Sheffield Crown Court - Yorkshire',
         courtType: 'CROWN',
```

Comment:
@mikehalmamoj I'd like to remove the `courtType` from the payload once you are not using it.